src/components/types/index.ts

```diff
@@ -1,9 +1,10 @@
-export interface IMyButtonProps {
+export interface MyButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   onClick: React.MouseEventHandler<HTMLButtonElement>;
   children: React.ReactNode;
   disabled?: boolean;
 }
 
-export interface IHelloProps {
-  title: string;
-}
\ No newline at end of file
+export interface HomeProps {
```

Comment:
@Krysthecoder @Cema2019 ❗ important: Using I before our interfaces is not a Typescript convention but a C# one. So let's try to keep this as a convention for this and future projects.